### ISSUE TITLE
Do not show snippets inside PHP code scope

### ISSUE THREAD

**Issue by mityukov** [wcwhitehead/bootstrap-3-snippets#46]
Is there a way to configure the extension so that it doesn't show inside "php code" scope?

Exmaple:

```
<html>

<?php

echo func_generate_header();
// here, inside the "code" scope, don't display bs3 snippets suggestions

?>

<body>

<!-- here, it can be useful, however -->

</body>
</html>
```

**Comment by sergejostir** [comment]
Not really, it's activated for file types only (as you can see in package.json).

I would also like to see some settings added to this package, so I can choose for which file types it activates. Currently you have to edit package.json manually (and re-apply after every update).
